azext_iot/_help.py

```diff
@@ -820,6 +820,31 @@
         --metric-type system
 """
 
+helps[
+    "iot hub configuration test-queries"
+] = """
+    type: command
+    short-summary: Validates target condition and custom metric queries for a configuration on the IoT Hub
+    long-summary: |
+      This command supports both inline json and file input for custom metric queries `--custom-metric-queries`.
+      Review examples and parameter description for details on how to fully utilize the operation.
+
+      - For bash inline json format use '{"key":"value"}' and \\ (backslash) for command continuation.
+      - For powershell inline json format use '{\\"key\\":\\"value\\"}' and ` (tilde) for command continuation.
+      - For cmd inline json format use \"{\\"key\\":\\"value\\"}\" and ^ (caret) for command continuation.
+      - For file based json input use "@/path/to/file". File based input avoids shell quotation issues.
```

Comment:
We should have this in a separate readme/file or wiki page that we link to the main page (and also any commands that we think needs it). 

Wanna write a "best practices for az cli beyond the given resources"? Let's start with adding this info (and any other weird quirks of the command shells) to the wiki page

Our wiki page to add to:
https://github.com/Azure/azure-iot-cli-extension/wiki/Tips

Leverage this doc (and or tell people to read this):
https://learn.microsoft.com/en-us/cli/azure/use-cli-effectively?tabs=bash%2Cbash2

Reply:
Also make sure you test these things out in a non powershell terminal - ex: ubuntu 

Google for how to add ubuntu to windows terminal
https://rkstrdee.medium.com/how-to-add-ubuntu-tab-to-windows-10s-new-terminal-271eb6dfd8ee

Reply:
updated and tested in multiple shells